src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -222,7 +223,7 @@ public class EntityConverter {
     }
 
 
-    private object? GetFieldValue(EntityInfo info, string name, TableEntity entity) {
+    private object? GetFieldValue(EntityInfo info, string name, TableEntity entity, int iterationCount = 0) {
```

Comment:
I'd make the parameter required so that it doesn't get accidentally omitted upon a recursive call